test/test_tree_2d_shallowwater.jl

```diff
@@ -0,0 +1,41 @@
+module TestExamples2DShallowWater
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2d_dgsem")
+
+@testset "Shallow Water" begin
+  @trixi_testset "elixir_shallowwater_ec.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_shallowwater_ec.jl"),
+      l2   = [0.9911807999456177, 0.7340358820001013, 0.7446688375175905, 0.5875351036989047],
+      linf = [2.0118414065026426, 2.9946841579032526, 2.6555844309739545, 3.0],
+      tspan = (0.0, 0.25))
```

Comment:
I guess you checked EC and well-balanced properties using the elixirs tested here?

Reply:
Yeah, the actual errors here are nonsense, but it passes EC and well-balancedness for the discontinuous bottom topography when you run it and look at the out.